CONTRIBUTING.md

```diff
@@ -35,10 +37,54 @@ For more information about CLAs, please check out Alex Russell’s excellent
 post, [“Why Do I Need to Sign
 This?”](https://infrequently.org/2008/06/why-do-i-need-to-sign-this/).
 
-## Slack
+### Slack
 
 We host a public Slack with a dedicated channel for contributors and
 maintainers of open source projects hosted by New Relic. If you are
 contributing to this project, you're welcome to request access to the
 \#oss-contributors channel in the newrelicusers.slack.com workspace. To request
 access, see https://newrelicusers-signup.herokuapp.com/.
+
+## PR Guidelines
```

Comment:
We don't really have much for this repo and I've barely touched it, so left simple for now w/ version language and test content.